Variables/Boolean operators/task.md

```diff
@@ -1,6 +1,13 @@
 ## Boolean operators
 
 Boolean is a type of value that can only be `True` or `False` . The `==` (equality) operator checks whether the two variables being compared are equal.  
+
+Other comparison operators:
```

Comment:
These operators are discussed in the following task. Are we sure we do need to display them twice?